src/linear_algebra/bilinear_form.lean

```diff
@@ -40,8 +41,9 @@ structure bilin_form (R : Type u) (M : Type v) [ring R] [add_comm_group M] [modu
 (bilin_add_right : ∀ (x y z : M), bilin x (y + z) = bilin x y + bilin x z)
 (bilin_smul_right : ∀ (a : R) (x y : M), bilin x (a • y) = a * (bilin x y))
 
+/-- A linear map with two arguments is a bilinear form. -/
```

Comment:
I feel like this could be confused with a map from `M × M`.
```suggestion
/-- A map with two arguments that is linear in both is a bilinear form -/
```

Reply:
Alternatively:
` A linear map to linear self-maps is a bilinear form`